lib/bullet/notification/base.rb

```diff
@@ -73,7 +73,7 @@ def klazz_associations_str
       end
 
       def associations_str
-        ":includes => #{@associations.map { |a| a.to_s.to_sym unless a.is_a? Hash }.inspect}"
+        ".includes(#{@associations.map { |a| a.to_s.to_sym unless a.is_a? Hash }.inspect})"
```

Comment:
Currently if `a.is_a? Hash` then `nil` will be rendered here. I would remove but wanted to ask. 

Also, maybe better to not do `#inspect` as `#includes` accepts a splat.

Reply:
@sshaw I don't remember why `a.is? Hash` is here, but you can remove it and see if any test failed.